### PR TITLE
Oe 746/add release steps to readme

### PR DESCRIPTION
**Pull request by oliviawongnyc**
Adds detailed release steps to the repo's README per ticket [OE-746](https://jira.nypl.org/browse/SIMPLY-4198).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,17 +87,28 @@ There are two protected branches in this repository: `staging`, and `production`
...
 - A link to the associated [NYPL Jira ticket](https://nypl.jira.org) in the description when possible.
 - QA testing label, options include `no-qa`, `pending-qa`, and `passed-qa`.
 
+Pull requests on feature branches generate their own deployment URLs using Vercel. QA engineers use these feature branches to test changes before approval and merging to `staging`.
+
+We try to move code from `staging` to `production` once every two weeks.
+
+Generally, we cut a release on the Thursday before the end of the sprint. We do this by completing the following steps.
+
+1. On the `staging` branch, review the "UNRELEASED CHANGES" in the CHANGELOG. Update the "Fix version/s" of the corresponding tickets in Jira with the new release number. Move any tickets to "Ready for Prod" that aren't already there.
```
Comment (samanthaandrews) [inline README.md:103]: I'm realizing there is a similar section titled "# Deploying to production with Github Actions" and we should combine/consolidate them.

File: README.md [hunk 2]
```diff
@@ -88,17 +87,28 @@ There are two protected branches in this repository: `staging`, and `production`
...
+2. Create a branch off `staging` with a name like `OE-730/release-4.7.21`. In the CHANGELOG, update "UNRELEASED CHANGES" to the new version number. Update the version number in `package.json`. Run `npm install` to update `package-lock.json`.
+3. Make a PR to merge these changes into `staging` and request reviews from the team.
```
Comment (samanthaandrews) [inline README.md:105]: We changed this recently. Now, all of us devs can actually push directly to the `staging` branch which eliminates the need for two PR reviews. So usually in this step I update the changelog, update the version number, and run npm install. Then push that directly to staging.

**Review by samanthaandrews — CHANGES_REQUESTED** [review]
Can we make the base branch for this `OE-729/monorepo` to make sure it gets into the right place in the new monorepo setup? Should end up being in `apps/oew/README.md`

File: README.md (hunk 1 above)
Comment (oliviawongnyc) [reply README.md:103]: ah. yes.

**Pull request closed without merging**